app/models/solidus_paypal_commerce_platform/payment_method.rb

```diff
@@ -23,6 +24,14 @@ def cart_partial_name
       "paypal_commerce_platform"
     end
 
+    def display_on_product_page
+      preferences[:display_on_product_page]
+    end
+
+    def product_partial_name
```

Comment:
Not sure, but maybe this should be `product_page_partial_name`…

Reply:
Yes, that makes more sense - the method above it is named `display_on_product_page` so we might as well keep that naming to be consistent. Thanks!